lib/Driver/Driver.cpp

```diff
@@ -821,6 +821,9 @@ Driver::buildCompilation(const ToolChain &TC,
       translateInputAndPathArgs(*ArgList, workingDirectory));
 
   validateArgs(Diags, *TranslatedArgList);
```

Comment:
@jrose-apple Can we add a TODO here to verify later which validations could be moved to toolchain specific?

Reply:
Yeah, good idea.